### PR TITLE
DO NOT MERGE: Test stability of t/ui/27-plugin_obs_rsync_status_details.t

### PR DESCRIPTION
**Pull request by Martchus**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -153,7 +153,7 @@ test-heavy:
 
 .PHONY: test-ui
 test-ui:
-	$(MAKE) test-with-database TIMEOUT_M=25 PROVE_ARGS="$$HARNESS t/ui/*.t" GLOBIGNORE="t/*tidy*:t/*compile*:$(unstables)"
+	$(MAKE) test-with-database RETRIES=50 STABILITY_TEST=1 TIMEOUT_M=25 PROVE_ARGS="$$HARNESS t/ui/27-plugin_obs_rsync_status_details.t"
```
Comment (okurz) [inline Makefile:156]: oops, that should have been `RETRY=50`, sorry about that :/

Comment (okurz) [reply Makefile]: fixed in https://github.com/os-autoinst/openQA/pull/3888/commits/137eec91b12b7de25318f5afdcd2ca94c6589bba

**Comment by Martchus** [comment]
Not sure at which attempt it failed but it is definitely not stable yet:

```
export PERL5OPT="-mJSON::PP -MDevel::Cover=-select_re,'^/lib',+ignore_re,lib/perlcritic/Perl/Critic/Policy,-coverage,statement,-db,cover_db_ui, -It/lib -I/home/squamata/project/t/lib -MOpenQA::Test::PatchDeparse";\
RETRY=50 timeout -s SIGINT -k 5 -v $((600 * (50 + 1) ))m tools/retry prove -l --harness TAP::Harness::JUnit --timer t/ui/27-plugin_obs_rsync_status_details.t
[17:58:49] t/ui/27-plugin_obs_rsync_status_details.t .. 183/? Bailout called.  Further testing stopped:  Wait limit of 60 seconds exceeded for "tr#folder_Proj2appliances .dirtystatuscol .dirtystatus", no change: published on 2021-05-05 18:00:27 +0000 (appliances)
FAILED--Further testing stopped: Wait limit of 60 seconds exceeded for "tr#folder_Proj2appliances .dirtystatuscol .dirtystatus", no change: published on 2021-05-05 18:00:27 +0000 (appliances)
```

**Comment by okurz** [comment]
Hooray for the test then :)

**Pull request closed without merging**